engine/src/test/java/org/camunda/bpm/engine/impl/cfg/ProcessEngineConfigurationTest.java

```diff
@@ -17,17 +17,61 @@
 package org.camunda.bpm.engine.impl.cfg;
 
 import static org.assertj.core.api.Assertions.*;
+import static org.junit.Assert.assertEquals;
+
+import org.camunda.bpm.dmn.engine.DmnEngine;
+import org.camunda.bpm.dmn.engine.impl.DefaultDmnEngineConfiguration;
+import org.camunda.bpm.dmn.engine.impl.spi.el.ElExpression;
+import org.camunda.bpm.dmn.engine.impl.spi.el.ElProvider;
 import org.camunda.bpm.engine.ProcessEngineConfiguration;
+import org.camunda.bpm.engine.impl.el.ElProviderCompatible;
 import org.junit.Test;
 
 public class ProcessEngineConfigurationTest {
 
   @Test
   public void shouldEnableStandaloneTasksByDefault() {
     // when
-    ProcessEngineConfigurationImpl engineConfiguration = (ProcessEngineConfigurationImpl) ProcessEngineConfiguration.createStandaloneProcessEngineConfiguration();
+    ProcessEngineConfigurationImpl engineConfiguration = (ProcessEngineConfigurationImpl) ProcessEngineConfiguration
+        .createStandaloneProcessEngineConfiguration();
 
     // then
     assertThat(engineConfiguration.isStandaloneTasksEnabled()).isTrue();
   }
+
+  @Test
+  public void shouldUseDefaultElProviderByJuelExpressionManager() {
+    // when
+    ProcessEngineConfigurationImpl engineConfiguration = (ProcessEngineConfigurationImpl) ProcessEngineConfiguration
+        .createStandaloneProcessEngineConfiguration();
+
+    engineConfiguration.init();
+    DmnEngine dmnEngine = engineConfiguration.getDmnEngine();
+
+    // then
+    ElProvider expected = ((ElProviderCompatible) engineConfiguration.getExpressionManager()).toElProvider();
+    assertEquals(expected, ((DefaultDmnEngineConfiguration) dmnEngine.getConfiguration()).getElProvider());
+  }
```

Comment:
This is actually already covered in `DmnEngineConfigurationTest#setProcessEngineElProviderByDefault`. Sorry for not having thought of that before.